app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -122,19 +119,15 @@ class ConversationViewModel @Inject constructor(
 
         conversationViewState = conversationViewState.copy(messageText = "")
         viewModelScope.launch {
-            // TODO what if conversationId is null???
-            conversationId?.let {
-                // TODO: Handle error case when sending message
-                sendTextMessage(it, messageText)
-            }
+            sendTextMessage(conversationId, messageText)
```

Comment:
Shouldn't we keep the TODO comment until we have decided what to do if `sendTextMessage()` fails??